apps/bettafish/src/app/pages/browse/search/search.component.ts

```diff
@@ -29,4 +44,21 @@ export class SearchComponent {
             this.loading = false;
         });
     }
+    
+    submitSearch() {
+        this.router.navigate([], {
+            relativeTo: this.route,
+            queryParams: { query: this.searchForm.controls.query.value },
+            queryParamsHandling: 'merge',
+        }).catch(() => {
+            this.alerts.error(`Something went wrong! Try again in a little bit.`);
```

Comment:
Should we start making more of an effort to communicate error information to the users? Something like
```TypeScript
}).catch((error) => {
    this.alerts.error(`Something went wrong. Details: ${error}`);
});
```
...or something to that effect?